doc/vim-lsp.txt

```diff
@@ -151,7 +151,7 @@ on pyls (https://github.com/palantir/python-language-server)
 
        augroup lsp_install
             au!
-            autocmd User lsp_enabled call s:lsp_install()
+            autocmd User lsp_enabled call s:on_lsp_buffer_enabled()
```

Comment:
Should be lsp_buffer_enabled

Reply:
Yes, will merge https://github.com/prabirshrestha/vim-lsp/pull/601